### PR TITLE
Create SUPPORT.md

### PR DESCRIPTION
**Pull request by austinzheng**
Thoughts or comments appreciated.

### CODE REVIEW

**Comment by jpsim** [comment]
From what I can tell from the [blog post](https://github.com/blog/2400-support-file-support) announcing this, the contents should be more concise than the multi-paragraph approach you've taken here. Care to trim that down to a 1-liner? /cc @aviwarner

**Comment by bdash** [comment]
Why do you think that, @jpsim? The document in question is simply linked to at the top of the [New Issue page], much like our existing guidelines for contributing.

**Comment by aviwarner** [comment]
This is great @austinzheng. I'll go ahead and replicate on our other public repos. I think we're OK with the multi-paragraph format due to how the support guidelines are presented (just clarified this with @jpsim).

**Comment by austinzheng** [comment]
Sounds good. I'm wondering if I've missed anything, or if there are other resources we do (or specifically do not) want to mention.

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+## Stack Overflow
+
+If you have questions about Realm you can ask them on Stack Overflow. We continually monitor the [`realm` tag](https://stackoverflow.com/tags/realm). Please also tag your question with `ios`, `swift`, or other tags as appropriate.
+
+When asking questions on Stack Overflow, please keep in mind Stack Overflow's [question guidelines](https://stackoverflow.com/help/how-to-ask), and please use their search functionality to see if your question has been asked before.
+
+## GitHub Issues
+
+If you are running into issues with Realm, including questions which are very specific to your particular application or potential bugs, we encourage you to file an issue on our [GitHub issue tracker](https://github.com/realm/realm-cocoa/issues). Please fill out the issue template in its entirety.
```
Comment (bdash) [inline SUPPORT.md]: I think It'd be worth pointing users to CONTRIBUTING.md in this section.

**Comment by aviwarner** [comment]
@austinzheng there's some fine tuning to be done here, but I'm ok with this going up in the short term as a v1. I'll work on a rewrite just to help clarify what we do/don't support and where certain channels are more appropriate.

**Pull request merged**